chef/cookbooks/network/templates/default/suse-cfg.erb

```diff
@@ -38,8 +38,14 @@ BONDING_MODULE_OPTS=<%=quote("mode=#{@nic.mode} miimon=#{@nic.miimon}")%>
 <% iface["slaves"].each_with_index do |slave,i| -%>
 BONDING_SLAVE_<%=i%>=<%=quote(slave)%>
 <% end -%>
+<% if iface["mtu"] > 1500 -%>
```

Comment:
I think it might be desireable to have it the same condition like the MTU= line below e.g. != 1500